app/api-v1/controllers/Recipe/index.js

```diff
@@ -0,0 +1,51 @@
+const { runProcess } = require('../../../utils/dscp-api')
+const { client } = require('../../../db')
+const { mapRecipeData } = require('./helpers')
+const { BadRequestError, NotFoundError } = require('../../../utils/errors')
+
+module.exports = {
+  transaction: {
+    create: async (req) => {
+      const service = '/recipe/{id}/creation'
+      const { id } = req.params
+      if (!id)
+        throw new BadRequestError({
+          message: 'missing [id] param from request',
+          service,
+        })
+
+      const [recipe] = await client.from('recipes').select('*').where({ id })
```

Comment:
`knex` client interface should not appear in controllers. We should export simple functions from the `db` that perform the required actions

Reply:
updated along wiith 231